src/Options/Applicative/Help/Pretty.hs

```diff
@@ -39,10 +103,25 @@ ifNotAtRoot f doc =
 --   group.
 groupOrNestLine :: Doc -> Doc
 groupOrNestLine =
-  Union
+  PPI.Union
     <$> flatten
-    <*> ifNotAtRoot (line <>) . nest 2
+    <*> ifNotAtRoot (line <>)
+  where flatten :: Doc -> Doc
+        flatten doc = case doc of
+          PPI.FlatAlt _ y     -> flatten y
+          PPI.Cat x y         -> PPI.Cat (flatten x) (flatten y)
+          PPI.Nest i x        -> PPI.Nest i (flatten x)
+          PPI.Line            -> PPI.Fail
+          PPI.Union x _       -> flatten x
+          PPI.Column f        -> PPI.Column (flatten . f)
+          PPI.WithPageWidth f -> PPI.WithPageWidth (flatten . f)
+          PPI.Nesting f       -> PPI.Nesting (flatten . f)
+          PPI.Annotated ann x -> PPI.Annotated ann (flatten x)
 
+          x@PPI.Fail   -> x
+          x@PPI.Empty  -> x
+          x@PPI.Char{} -> x
+          x@PPI.Text{} -> x
```

Comment:
Why is this inlined?

Reply:
What did you mean by inlined?

Reply:
`flatten` came from the deprecated `ansi-wl-pprint` library's internal module.  `prettyprinter` doesn't export such a function.

The function is defined in the `where` clause because this is the only function that needs to use it.